packages/lib-user/src/components/MyGroups/MyGroups.js

```diff
@@ -1,25 +1,27 @@
-import { Grid, ResponsiveContext } from 'grommet'
+import { Grid } from 'grommet'
 import { node } from 'prop-types'
-import { useContext } from 'react'
+import styled from 'styled-components'
+
+const StyledGrid = styled(Grid)`
+  gap: 20px 40px;
+  justify-items: center;
+  grid-template-columns: 1fr 1fr;
+
+  @media (width < 80rem) {
```

Comment:
Towards the higher end of this range there's a lot of empty horizontal space around the group cards, maybe could be lower (70-ish em?), but this can easily be revisited if there's similar feedback.

Reply:
Agreed there's a bit of an awkward layout around 60-80rem because of the space around group cards. I'll still with 80rem for now though because smaller screen widths start smushing the group cards together with overlapping text.